capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -117,10 +119,17 @@ class ClubPage extends Component {
 
   render() {
     const isOwner = this.state.owner && this.state.club.ownerID === window.localStorage.getItem('userID');
-    const bookTile = this.state.book.authors && <BookSearchTile book={this.state.book} bookLists={this.props.bookLists} updateBookLists={this.props.updateBookLists} />;
+    const bookTile = this.state.book.authors 
+        ? <BookSearchTile book={this.state.book} bookLists={this.props.bookLists} updateBookLists={this.props.updateBookLists} />
```

Comment:
Could you break up the props into multiple lines to make it more clear?

Reply:
Sure!